account_move_report/views/account_reports.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="paperformat_account_move_report" model="report.paperformat">
+        <field name="name">Account moves report</field>
+        <field name="default" eval="True" />
+        <field name="format">Letter</field>
+        <field name="orientation">Landscape</field>
+        <field name="margin_top">40</field>
+        <field name="margin_bottom">12</field>
+        <field name="margin_left">5</field>
+        <field name="margin_right">5</field>
+        <field name="header_line" eval="False" />
+        <field name="header_spacing">40</field>
+        <field name="dpi">90</field>
+    </record>
```

Comment:
Use the default paperformat, it is not necessary use a new paper format per report we have a global one in the company.